Userland/Libraries/LibJS/Bytecode/Pass/LoadElimination.cpp

```diff
@@ -182,15 +182,14 @@ void EliminateLoads::perform(PassPipelineExecutable& executable)
 {
     started();
 
-    // FIXME: If we walk the CFG instead of the block list, we might be able to
-    //        save some work between blocks
-    for (auto it = executable.executable.basic_blocks.begin(); it != executable.executable.basic_blocks.end(); ++it) {
-        auto const& old_block = *it;
+    auto cfg = executable.cfg.release_value();
```

Comment:
Make sure we generate a CFG before just releasing it
For that look into the PassManager and where it is created and what other passes do